CHANGELOG.md

```diff
@@ -20,6 +20,13 @@ which will generate a report using the supplied metrics build (which must be cap
 - Other available report commands are similar to batches: `get`, `wait`, `logs`.
 - For full details of how reports work and how to generate a report, please read the main ReSim [docs](https://docs.resim.ai)
 
+### v0.3.3 - June 20 2024
```

Comment:
i would just add this to the unreleased section right now?

Reply:
Unreleased since CLI releases are done in a different flow? Or just because the user information isn't visible (yet) in the webapp?